cm_sdk/api/cm_api.py

```diff
@@ -0,0 +1,133 @@
+from cm_sdk.client import CloudManagerClient
+from cm_sdk.models.alert_config import alert_config
+from cm_sdk.models.host import host
+from cm_sdk.models.backup_config import backup_config
+
+class Api(object):
```

Comment:
You should probably expose this class via `__all__` in this dir's `__init__.py` to simplify the user experience (it'd allow them to do `from cm_sdk.api import Api`